tools/autograd/derivatives.yaml

```diff
@@ -1055,6 +1049,12 @@
 - name: view(Tensor(a) self, int[] size) -> Tensor(a)
   self: grad.reshape(self.sizes())
 
+- name: view_as_real(Tensor(a) self) -> Tensor(a)
+  self: at::view_as_complex(grad.contiguous()).conj() # gx0 - i gx1
```

Comment:
BTW, does this imply our imag gradient from before was wrong? Because it seems like going through `view_as_real` to get the imag part would have skipped the negation

Reply:
I am not sure what you mean by if our imag gradient was wrong. But what's happening here is:

```
f: R^2 -> C
vjp = [gx, -gy] * [[1, 0], [0, 1]]
```